### DISCUSSION TITLE
Growatt SPH6000 data

### DISCUSSION THREAD

**Discussion by Peter86000** [egguy/grott-home-assistant-add-on#58] [Q&A]
Hi,
I'm using "Grott" addon, it works perfectly for me, thank you for this addon.
It seems that oly a part of Growatt SPH6000 data are retrieved, and I need the others (SOC, ...). How can I catch those other data ? 

Thank you for your replies !

**Comment by Neflardio** [comment]
Easiest way to figure out what is going on is to post your logs.

It sounds like inverter type might not be set to 'sph' but can't know for sure without logs.

**Reply by Peter86000** [reply]
Hi, 
I was using Grott Edge with 'sph' configuration. I now use Grott and all datas are displayed. 
1 problem remains : none of datas required with "Energy" dashboard can be selected : pv1_watt, import_from_grid, ...

**Reply by egguy** [reply]
Do you even seen the total counter? You can only use the total counter in the energy dashboard.
